sdks/java/io/hadoop-input-format/src/test/resources/kubernetes/cassandra/data-load.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+# Identify the pod
+cassandra_pods="kubectl get pods -l name=cassandra1"
+running_seed=$($cassandra_pods --no-headers | \
+    grep Running | \
+    grep 1/1 | \
+    head -1 | \
+    awk '{print $1}')
+echo "$running_seed"
+cassandra_svc="kubectl get svc cassandra1"
+echo
+
+sleep 2m
```

Comment:
When "kubectl get svc cassandra1" is executed, it creates a service for cassandra. But the external IP for the service gets generated after couple of minutes. Till it gets generated, it shows "<pending>" for the IP address. Hence, we have added sleep. Please let us know your view on this.